client/apollo/react/src/Form/Checkbox/Checkbox/CheckboxCommon.tsx

```diff
@@ -1,13 +1,13 @@
 import React, { forwardRef } from "react";
 import { Svg } from "../../../Svg/Svg";
 
-export type CheckboxProps = React.InputHTMLAttributes<HTMLInputElement> & {
+export type CheckboxProps = {
   errorId?: string;
   hasError?: boolean;
   className?: string;
 } & Omit<React.InputHTMLAttributes<HTMLInputElement>, "disabled">;
```

Comment:
pourquoi omit la prop disabled?
```suggestion
export type CheckboxProps = {
  errorId?: string;
  hasError?: boolean;
} & React.ComponentPropsWithRef<'input'>;
```

Reply:
Aucune idée, c'était là de base et je ne voulais pas faire de régression.
Mais sinon je suis d'accord qu'on ne devrait pas enlever cette prop.

Reply:
(@abderrahman-fawzy-acn-axa je te laisse resolve ou continuer la discussion en fonction de ce que tu en penses)